cartridges/int_algolia_sfra/cartridge/static/default/js/algolia/insights-config.js

```diff
@@ -91,6 +91,51 @@ function enableInsights(appId, searchApiKey, productsIndex) {
         });
     }
 
+    // Purchase events.
+    // The purchase event is sent when the user reaches the Order-Confirm endpoint.
+    // We can detect it because it's the only endpoint that sets a `orderUUID` in `pdict` properties:
+    // https://github.com/SalesforceCommerceCloud/storefront-reference-architecture/blob/dec9c7c684275127338ac3197dfaf8fe656bb2b7/cartridges/app_storefront_base/cartridge/controllers/Order.js#L87
+    // Our 'algolia-insights' tag exposes its "data-order" attribute only in that case. When it is present, we can send the purchase event.
+    //
+    // TODO: keep track of the queryID in the local storage when users give their consent, and send a 'purchasedObjectIDsAfterSearch' event
+
+    const insightsData = document.querySelector('#algolia-insights');
+    const order = insightsData.dataset.order;
```

Comment:
Great PR thanks! We may only put a null check here, but not mandatory. I am approving PR without it, too!

Reply:
Doesn't cost anything to add it, we never know :ok_hand: